tensorboard/plugins/debugger_v2/debug_data_multiplexer.py

```diff
@@ -48,10 +81,14 @@ def run_in_background(target):
     Args:
       target: The target task to run in the background, a callable with no args.
     """
-    # TODO(cais): Implement repetition with sleeping periods in between.
     # TODO(cais): Add more unit tests in debug_data_multiplexer_test.py when the
     # the behavior gets more complex.
-    thread = threading.Thread(target=target)
+    def _run_repeatedly():
+        while True:
+            target()
+            _timer.sleep()
+
+    thread = threading.Thread(target=_run_repeatedly)
```

Comment:
Existing code, but shouldn't this be a daemon thread?  https://docs.python.org/3.6/library/threading.html#threading.Thread.daemon

Otherwise if this thread isn't terminated properly at program exit it could prevent TensorBoard from exiting.

Reply:
Done.